doc/manual/customizing.rst

```diff
@@ -49,49 +89,202 @@ inherit them.  You can also manually copy the example file from
+The optional third and following items in a list are specifications for
+nested content.  The format for nested content depends on the type of the
+container:
+
+* ``seq``, ``hpanel`` and ``vpanel`` types expect the nested content items to
+  be lists, as described above.
+* ``hbox`` and ``vbox`` content items can be lists (as described above) or
+  ``dict`` s. A ``vbox`` ``dict`` should have a ``row`` value and optionally a
+  ``stretch`` value; an ``hbox`` ``dict`` should have a ``col`` value and
+  optionally a ``stretch`` value.  The ``row`` and ``col`` values should be
+  lists as described above.
+* The ``ws`` (workspace) type takes one optional content item, which
+  should be a sublist containing 2-item lists (or tuples) with the format
+  ``(name, content)``, where ``content`` is a list as described above.  The
+  ``name`` is used to identify each content item in the way appropriate
+  for the workspace type, whether that is a notebook tab, MDI window
+  titlebar, etc.
+
+Here is the standard layout, as an example:
+
+.. literalinclude:: ../../ginga/examples/layouts/standard/layout
+    :language: python
+
+In the above example, we define a top-level window consisting of a vbox
+(named "top") with 4 layers: a hbox ("menu"), hpanel ("hpnl"), a
+workspace ("toolbar") and another hbox ("status").  The main horizontal
+panel ("hpnl") has three containers: a workspace ("left"), a vbox
+("main") and a workspace ("right").  The "left" workspace is
+pre-populated with an "Info" tab containing a vertical panel of two
+workspaces: "uleft" and "lleft" (upper and lower left).  The "right"
+workspace is pre-populated with a "Dialogs" tab containing an empty
+workspace.  The "main" vbox is configured with four rows of workspaces:
+"channels", "cbar", "readout" and "operations". 
+
+.. note:: The workspace that has as a configuration option ``default:
+          True`` (in this example, "channels") will be used as the
+          default workspace where new channels should be created.
+
+
+.. _sec-pluginconfig:
+
+==============================
+Customizing the set of plugins
+==============================
+
+In the configuration directory, the presence of a file ``plugins.json``
+will override the built-in configuration of plugins.  The file format is
+a JSON array containing JSON objects, each of which configures a plugin.
+Example:
+
+.. code-block::
+
+    [
+        {
+            "__bunch__": true,
+            "module": "Info",
+            "tab": "Synopsis",
+            "workspace": "lleft",
+            "start": true,
+            "hidden": true,
+            "category": "System",
+            "menu": "Info [G]",
+            "ptype": "global"
+       },
+       ...
+   ]
+    
+                 
+The keys for each object are defined as follows:
+
+* ``__bunch__``: should be present and set to ``true`` to force
+  deserialization to a ``~ginga.misc.Bunch.Bunch``.
+* ``module``: The name of the module in the ``$PYTHONPATH`` containing
```

Comment:
Is it only PYTHONPATH? What about those in `~/.ginga/plugins`?

Reply:
`~/.ginga/plugins` is added to the  `PYTHONPATH`. I think this is explained a little further in the document.

Reply:
Ah, okay. I see it at https://ginga--763.org.readthedocs.build/en/763/manual/customizing.html#custom-plugin-directory . Thanks!